invisible-plugin-sample/README.md

```diff
@@ -0,0 +1,3 @@
+# UXP Invisible Plugin Sample
+
+This Plugin is an example of creating invisible plugin in UXP. An invisible plugin remains hidden and never becomes visible during its whole life cycle. It means that these kind of plugins dont have any UI or Menu Items.
```

Comment:
Swap for this - "This plugin is an example of an invisible UXP plugin. An invisible plugin remains hidden during its whole life cycle -- meaning it does not have any UI or (persistent) menu options."

Reply:
@amandahuarng Done